integration_tests/configs/cosmovisor.jsonnet

```diff
@@ -0,0 +1,31 @@
+local config = import 'default.jsonnet';
+
+config {
+  'cronos_777-1'+: {
+    'app-config'+: {
+      'minimum-gas-prices': '5000000000000basetcro',
+      'json-rpc'+: {
+        api:: super['api'],
```

Comment:
no need to override `api` field?

Reply:
`::` mean remove, no api under [json-rpc](https://github.com/crypto-org-chain/cronos/blob/main/integration_tests/configs/cosmovisor.yaml#L9) before

Reply:
I see 👍